src/test_suite.rs

```diff
@@ -67,22 +70,25 @@ impl Test {
             }
             Box::new([wasmtime::Val::I32(0)])
         });
+        let elapsed = now.elapsed();
+        // Convert the elapsed time to milliseconds
+        // Seems hacky, might need refactoring
+        let elapsed_in_ms = elapsed.as_secs_f32() * 1000.0;
+
         logging::sub_indent();
         let logs = logging::flush();
 
+        let msg = format!("{} - {:.3?}ms", self.name.clone(), elapsed_in_ms);
```

Comment:
It would be cool to print `elapsed_in_ms` in a different color, so it's more noticeable in the logs. :)